### ISSUE TITLE
Stereo support

### ISSUE THREAD

**Issue by andrey-zakharov** [fabmax/kool#21]
At least, to be able handle stereo audio, [Stereo](https://developer.mozilla.org/en-US/docs/Web/API/StereoPannerNode) should be wrapped from JS side, and Java's Mixer from JVM.

thinking about another one `AudioOutput` class?

**Comment by fabmax** [comment]
Yes, audio support is still very basic. Stereo sound would be nice.

**Comment by andrey-zakharov** [comment]
Issue that AudioOutput "too early" goes to `generate`. I mean 
 - on js (and jvm prob.) side `StereoPanNode` should be connected to `AudioContext`, but we have 
 - on lib side: `AudioOutput` with `MixNode.generate` tandem.
